### DISCUSSION TITLE
Cloud visits could be real people?

### DISCUSSION THREAD

**Discussion by eurobank** [CIDRAM/CIDRAM#291] [General]
Hello.

If anybody knows: could visitors blocked as Cloud, be real people?

Or they are 100% bots, scripts or other strickly automated services?

**Comment by Maikuolan** [comment]
In theory, visitors blocked as cloud should not be real people, and by far, the *vast majority* of such visits should always be just bots, scripts, automated services, etc, and I always try to avoid as best as possible classifying something as cloud if/when we're able to identify potential human endpoints at the addresses/ranges in question.

However, given how quickly ranges are bought/sold between different networks nowadays, and thus how quickly signatures can become stale due to that buying/selling/transfer of ranges between networks, given the risk of false positives (which, we always aim to minimise as close to zero as possible, but of course, very few risks in life can ever be truly brought down to absolute zero, and the risk of false positives here is no exception to that), given the risk of human error occurring during the process of analysing/investigating networks, the sources of various unwanted traffic and so on, and also given the ever-increasing fragmentation of the intended purposes of various networks nowadays and as time goes on (e.g., a network originally intended just as a domestic ISP suddenly starts offering streaming services or webhosting services as an add-on for their customers, or a dedicated cloud provider, server farm, etc suddenly starts offering corporate connectivity, businesses buy into that, couldn't be bothered paying for separate lines for their normal internet usage, start routing their normal internet usage through corporate lines, etc, all causing some networks to be carrying all kind of completely different traffic), it's basically impossible to 100% guarantee anything.

With that in mind.. ~99% of blocked "cloud" requests should *not* be from real people, but there's always that ~1% chance that they *will*, in fact, be requests from real people.

Whenever we encounter such a thing, what we need to do is reassess the various risks associated with blocking (or not blocking) the addresses/ranges/networks in question, and weigh up the pros and cons. Regardless of our reasons for blocking that network in the past, do we, at the present moment, continue to have a valid reason for blocking that network? Are we still seeing dangerous/unwanted traffic from that network, or do we have legitimate reason to belief that we're going to see such traffic from that network in the future (thus warranting that the network in question remains blocked)? How often do we see human traffic or any other legitimate/wanted forms of traffic from that network? Comparatively, which do we see more, between dangerous/unwanted traffic and legitimate/unwanted traffic? For blocking that network, do the pros outweigh the cons, or not? Or, are there any further measures we could take to reduce the risk of blocking legitimate traffic while still continuing to block unwanted traffic?

The circumstances and considerations which contribute to the decision of whether to block some addresses/ranges/networks is likely going to periodically change, and therefore, a decision which is correct today, might be incorrect in half a year, a year, or a few years from now, and visa versa, or the exact way in which something is blocked might need to be changed over time, too (e.g., perhaps one moment we classify something as cloud, but maybe need to classify it as something else later in the future, or visa versa).

As annoying as it may sometimes be, if we want to ensure that the decisions we make at any particular present moment are as correct as they can possibly be, we'll inevitably be forced to also reassess our old decisions from the past, to ensure that they're still applicable/relevant/correct for the present moment, too. This affects me too, in terms of for when I update the default signature files, as well as others, in terms of whether they maybe need to write bypasses for specific networks for their specific websites, auxiliary rules specific to their websites and so on. It's an ongoing process over an always-evolving medium, I think. :-)

**Reply by eurobank** [reply]
Thanks for the reply.

So the risk is not because a cloud visit can be real people but from a mistake in network included as cloud? IP error that is.
